include/swift/ABI/ObjectFile.h

```diff
@@ -126,6 +131,8 @@ class SwiftObjectFileFormatCOFF : public SwiftObjectFileFormat {
       return ".sw5prtc$B";
     case protocs:
       return ".sw5prt$B";
+    case mpenum:
+      return ".sw5mpen";
```

Comment:
I think that this here is the issue.  The `$B` is super critical.  It actually is used to coalesce the data between the begin and end makers.  This works with https://github.com/apple/swift/blob/main/stdlib/public/runtime/SwiftRT-COFF.cpp to ensure that the runtime is able to correctly locate the data.  You will also need to update the file to indicate a new metadata section.  There is a peer of it for ELF as well.

Reply:
I took a guess and updated these files.  Does this look right now?

Reply:
Yeah, that seems about right.